views/forms/ConfigAssessmentFormFields/index.jsx

```diff
@@ -30,11 +34,13 @@ const ConfigAssessmentFormFields = ({
         label="Category"
         fullWidth
       />
-      <TextInput
+      <PredictiveInputSelect
         control={control}
         name={`config.${index}.analysis`}
+        onChange={handleAssessmentSearch}
+        onBlur={handleClear}
```

Comment:
why are we clearing on blur?

Reply:
As a user if I fill out on configuration card, and then I add another card and search for an assessment, I thought that in this case it would be best to start fresh with a new list of options. 

I think we might want to wait for user feedback on this, in case they might want to see the list from the previous search.